index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+	<script src="https://kit.fontawesome.com/27ebd9ab14.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<header>
+	<nav>
+		<div class="header-page">
+			<a href="#index.html" class="Mylogo"> My Logo</a>
+			<a class="Hamberger"><i class="fa-solid fa-bars"></i></a>
+		</div>
+	</nav>
+	
+		<div class="header-container">
+		  <h1 class="title1">I'm Gardimy <br>Glad to see you!</h1>
+		  <p class="title2">
+			I'm a software developer! I can help you build a product , 
+			feature or website Look through some of my work and experience! If you 
+			like what you see and have a project you need coded,
+			 don't hestiate to contact me.
+		  </p>
+
+		  <div class="social-icon">
+			<p class="connect" href="#">LET'S CONNECT</p>
```

Comment:
- [ OPTIONAL] As required for this project The subtitle (if existing) should use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`, kindly remove the `p` tag here and use header tag for the screen readers reason :wink:  And  I just noticed that your design has some issue check this :point_down: 
![Screenshot from 2023-03-23 03-58-30](https://user-images.githubusercontent.com/88894525/227082748-6e37056e-de8f-4699-859c-7ff5ac5da0b3.png)